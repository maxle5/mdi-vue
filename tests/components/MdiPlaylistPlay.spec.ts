
import { mount } from "@vue/test-utils";
import { expect, test } from "vitest";
import MdiPlaylistPlay from "../../src/components/MdiPlaylistPlay.vue";

test("MdiPlaylistPlay snapshot", () => {
  const wrapper = mount(MdiPlaylistPlay, {});
  expect(wrapper.html()).toMatchSnapshot();
});
