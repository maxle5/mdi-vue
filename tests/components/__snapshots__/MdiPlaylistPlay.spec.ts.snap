// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`MdiPlaylistPlay snapshot 1`] = `
"<svg width="24" height="24" viewBox="0 0 24 24">
  <path d="M3 10H14V12H3V10M3 6H14V8H3V6M3 14H10V16H3V14M16 13V21L22 17L16 13Z" style="fill: currentColor;"></path>
</svg>"
`;
